tests/test_api_security.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the CAG API ViewSets.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import pytest
+from django.views import View
+from course_access_groups import views as cag_views
+from course_access_groups.security import CommonAuthMixin
+
+
+class TestCourseAccessGroupsViewSet(object):
+    """
+    Tests for the CourseAccessGroupsViewSet APIs.
+    """
+
+    view_classes = [
+        variable for variable in [
+            getattr(cag_views, var_name)
+            for var_name in dir(cag_views)
+        ]
+        if (isinstance(variable, type) and issubclass(variable, View))
+    ]
+
+    def test_sanity_check(self):
+        """
+        Ensure that `view_classes` contains the correct classes.
+        """
+        assert self.view_classes, 'View classes are being found correctly.'
+        assert cag_views.CourseAccessGroupViewSet in self.view_classes
+
+    @pytest.mark.parametrize('view', view_classes)
```

Comment:
This is cool, however, it is also opaque. I would prefer to have the view classes be explicit to make it clear what is being tested. Looking at this, I have no idea

Reply:
Added documentation to ensure it's clearer.